app/controllers/application_controller.rb

```diff
@@ -20,6 +20,12 @@ def pundit_scope
     Pundit.policy_scope(current_user, resource)
   end
 
+  rescue_from ActiveRecord::RecordNotUnique do |error|
+    render json: {
+      errors: "Duplicate record: #{error.message[/Key \(.+\).+\./]}"
+    }, status: :conflict
+  end
```

Comment:
This is a possibility for any controllers using ActiveRecord `import!`, though we really never expect it for `update_rules` since we provide a specific list of rules, those that don't exist on the profile already and are in the profile's benchmark. Still, this rescue may prove useful to another controller.